### ISSUE TITLE
Reinstalling scoop fails after a windows reinstall

### ISSUE THREAD

**Issue by alexwhittemore** [lukesampson/scoop#1930]
A few days ago, my bluetooth stack decided it was no longer able to remove devices, and the only fix was a reinstall of windows (Yes, windows made me reinstall due to a mouse that wouldn't connect!). Anyway, I took the slightly less disruptive route of "keep my documents and data", which mostly leaves the C:\Users folder in-tact.

The net impact is that, for whatever reason, `>installed scoop` returns `True`, even though scoop isn't actually installed. I have no idea how to go about convincing powershell it is, in fact, not installed. 

Edit: I should be clear, C:\Users\me\appdata\local\scoop does not exist.

**Comment by Ghasan** [comment]
scoop normally installs inside your home folder: C:\Users\me\scoop. What is "installed" command here, I did not see it before?

**Comment by alexwhittemore** [comment]
That's EXACTLY the info I needed - for some reason, I couldn't figure that out at ALL; every reference in issues seems to be to appdata\local\scoop. I deleted C:\Users\me\scoop and .scoop, and now the installer works. 

I spent a bit of time trying to figure out WTF "installed" was as well but couldn't find ANYTHING. Turns out, that's because it's a function (or cmdlet or something? I don't know powershell...) that gets created during the install script. So it only exists after you try to run it. 

Thanks again!
